cmd/server/handler.go

```diff
@@ -226,14 +258,27 @@ func setup(d driver.Driver, cmd *cobra.Command) (admin *x.RouterAdmin, public *x
 				driver.MetricsPrometheusPath,
 				"/",
 			},
+			WriteKey:     "h8dRH3kVCWKkIFWydBmWsyYHR4M0u0vr",
 			BuildVersion: d.Registry().BuildVersion(),
 			BuildTime:    d.Registry().BuildDate(),
 			BuildHash:    d.Registry().BuildHash(),
-		},
-	)
+		}
+		b, err := metricsx.NewFormattedSegmentBridge(ctx, s, l, d.Registry().PrometheusManager().Registry)
```

Comment:
Ok I see - so segment actually works (mostly) independent of Prometheus except for e.g. memory stats?